src/main/java/org/galatea/starter/utils/clock/ClockWrapper.java

```diff
@@ -0,0 +1,25 @@
+package org.galatea.starter.utils.clock;
+
+import java.time.Clock;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+
+public class ClockWrapper {
```

Comment:
You have the basic idea down, but the implementation could use some changes.

First, some potential issues:
1. The `clock` variable here is static but non-final and can and does get modified. This can lead to unexpected behavior when someone calls `getClock()`, because someone else might have changed which Clock is being used. For example, `testGetHistoricalPriceNoRangeWithDate()` sets the Clock to a fixed clock, but then never changes it back - meaning any tests that run after that one would continue using the fixed clock, and it can be hard to figure out why that's happening after the fact.
2. This class is available in the main code, but `useFixedClockAt()` should only ever be used in tests.
3. IexRestController calls `getClock()` only once, when it's instantiated. In our IexRestControllerTest, I'm pretty sure IexRestController only gets instantiated once, and then that same single instance is reused across all the tests. So even if we modify the ClockWrapper clock, the IexRestController clock would still be the clock it got back when it first got instantiated.

A better way would be to directly override the clock used in IexRestController rather than having this ClockWrapper class. There are two common ways to do this during tests:
1. Make use of dependency injection: just like we inject an actual IexService during normal application running, but replace that with a mock during tests, we can inject a Clock that can get replaced with a mock during tests
2. Manually set the clock in IexRestController with `private final Clock clock = Clock.systemDefaultZone()` or something similar, and then overwrite it with reflection during individual tests (such as with Spring's `ReflectionTestUtils`).

Both of these are a bit easier when we create a new instance of the class under test (IexRestController) before each test, which happens in some other tests like SettlementServiceTest. However, in IexRestControllerTest, the class is created once and then reused, as mentioned above. So, for both the suggested alternatives, we would want to reset the clock to its original value before each test, which could be accomplished with a `@Before` method (see SettlementServiceTest for an example of `@Before`).